### PR TITLE
Match tooltip styles to VS Code

### PR DESCRIPTION
**Pull request by mattseddon**
Closes https://github.com/iterative/vscode-dvc/issues/2254 by matching tooltip styling to VS Code's native trees.

### Demo

https://user-images.githubusercontent.com/37993418/189028120-a7538d70-d583-4975-a8d3-eadd8d844526.mov

Screen Shot 2022-09-08 at 1 32 46 pm

- add a border to tooltips
- remove arrows from tooltips and context menus => Remove the triangle (on top) from the context menus from #2241 
- remove unnecessary code and styles

### CODE REVIEW

**Comment by mattseddon** [comment]
@julieg18 looks like the truncation tooltip for experiment names isn't working anymore.

**Review by shcheklein — APPROVED** [review]

**Comment by julieg18** [comment]
> @julieg18 looks like the truncation tooltip for experiment names isn't working anymore.

I'm not sure what you mean 🤔  I've never seen a tooltip appear for experiment names (that I can remember 😅): 

image

What is the behavior supposed to be for experiment names?

File: webview/src/shared/components/tooltip/styles.module.scss [hunk 1]
```diff
@@ -30,3 +26,17 @@ $font: var(--vscode-font-family);
     z-index: 2;
   }
 }
+
+.tooltipContent {
+  border: 1px solid $tooltip-border;
+  white-space: pre-wrap;
+  padding: 5px;
```
Comment (julieg18) [inline webview/src/shared/components/tooltip/styles.module.scss:33]: Should we decrease the plots info tooltips padding now that they already get some padding from `.tooltipContent`:

image

**Review by julieg18 — APPROVED** [review]
Great work!

**Review by wolmir — APPROVED** [review]

File: webview/src/shared/components/tooltip/styles.module.scss (hunk 1 above)
Comment (mattseddon) [reply webview/src/shared/components/tooltip/styles.module.scss:33]: I changed the padding [here](https://github.com/iterative/vscode-dvc/pull/2353/files#diff-71037f545ae5c406885ca4620eaf6f314bcc77c43842d017d318cc958723c11cR368).

**Review by wolmir — APPROVED** [review]

**Comment by mattseddon** [comment]
> > @julieg18 looks like the truncation tooltip for experiment names isn't working anymore.
> 
> I'm not sure what you mean 🤔 I've never seen a tooltip appear for experiment names (that I can remember 😅):
> 
> image
> 
> What is the behavior supposed to be for experiment names?

I remember that are some point the branch name (maybe) would show a tooltip when it was truncated... might have dreamed it.

**Pull request merged**